ci/pipelines/concourse-pypi-resource-pipeline-template.yml

```diff
@@ -171,33 +160,35 @@ jobs:
     passed: [concourse-pypi-resource-build]
   - get: pypi.git
   - get: python.docker
-  - get: pypi-wheel-example
-    trigger: false
   - get: version
-    params:
-      bump: "minor"
   - put: cogito
     inputs: [pypi.git]
     params: {state: pending}
   - task: build-example-wheel
     image: python.docker
-    file: pypi.git/ci/tasks/build-example-wheel.yml 
-  - put: version
-    params: {file: version/version}
+    file: pypi.git/ci/tasks/build-example-wheel.yml
   - put: pix4d-pypi
     inputs:
       - dist
     params:
       glob: 'dist/*'
 
 - name: publish-concourse-pypi-resource
+  <<: *base-job
   plan:
+  - get: pypi.git
+  - get: version
+    params:
+      bump: patch
   - get: pypi.dev-docker
```

Comment:
are we sure that we are using the image that was just produced in step before?

Reply:
I'd say yes. The output of the job that creates the Docker image is the input of the acceptance test.
you can check for example:
- https://builder.ci.pix4d.com/teams/developers/pipelines/concourse-pypi-resource-pipeline-fix-pipeline/jobs/concourse-pypi-resource-build/builds/9
- https://builder.ci.pix4d.com/teams/developers/pipelines/concourse-pypi-resource-pipeline-fix-pipeline/jobs/pypi-acceptance-test/builds/9

and check the same sha is used:
```
sha256:55dccffbd2305fc40640cbd488a4a699f97272b9906facab77b5f4a38df8d771
```